src/plot_api/plot_api.js

```diff
@@ -245,9 +245,7 @@ exports.plot = function(gd, data, layout, config) {
                 .attr('height', fullLayout.height);
         }
 
-        return Lib.syncOrAsync([
-            subroutines.layoutStyles
-        ], gd);
```

Comment:
Too funny, so it just works! We may never know what obviated this...

Reply:
_just works_<sup>TM</sup>